### PR TITLE
Break: Finish migration of health libraries to witchcraft-go-health

### PR DESCRIPTION
**Pull request by asanderson15**
## After this PR

==COMMIT_MSG==
Finish migration of health libraries to witchcraft-go-health
==COMMIT_MSG==

## Possible downsides?

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/witchcraft-go-server/227)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -12,10 +12,10 @@ require (
 	github.com/palantir/pkg/metrics v1.0.1
 	github.com/palantir/pkg/objmatcher v1.0.1
 	github.com/palantir/pkg/safejson v1.0.1
-	github.com/palantir/pkg/safeyaml v1.0.1
 	github.com/palantir/pkg/signals v1.0.1
 	github.com/palantir/pkg/tlsconfig v1.0.1
 	github.com/palantir/witchcraft-go-error v1.4.3
+	github.com/palantir/witchcraft-go-health v0.0.0-20201110010813-1ffc140553de
```
Comment (asanderson15) [inline go.mod]: We should release a tagged version of WGH and bump to depend on that before merging this.

**Review by nmiyake — CHANGES_REQUESTED** [review]
Minor question about release notes. Otherwise, content looks good, so I think we're good to cut a release of WGH and then update this PR as well.

> Reviewed 57 of 57 files at r1.
> *[Reviewable](https://reviewable.io/reviews/palantir/witchcraft-go-server/227#-:-MLn7LoyEhzWULOMXlR_:b-ygk46i)* status: all files reviewed, 2 unresolved discussions (waiting on @asanderson15)

___
*[changelog/@unreleased/pr-227.v2.yml, line 3 at r1](https://reviewable.io/reviews/palantir/witchcraft-go-server/227#-MLn6uXNDOvJyre-LRM4:-MLn6uXNDOvJyre-LRM5:beq7im1) ([raw file](https://github.com/palantir/witchcraft-go-server/blob/aab0c5aad3f41ba92996db5a74e0d19abfa92500/changelog/@unreleased/pr-227.v2.yml#L3)):*
> ```YAML
> type: break
> break:
>   description: Finish migration of health libraries to witchcraft-go-health
> ```

Will this entry/description make sense as part of the over-all release notes of v2.0?

**Review by asanderson15 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/palantir/witchcraft-go-server/227)* status: 56 of 57 files reviewed, 2 unresolved discussions (waiting on @nmiyake)

___
*[changelog/@unreleased/pr-227.v2.yml, line 3 at r1](https://reviewable.io/reviews/palantir/witchcraft-go-server/227#-MLn6uXNDOvJyre-LRM4:-MLnAEQuEkRQ9oGFj7SE:b-ydm9vj) ([raw file](https://github.com/palantir/witchcraft-go-server/blob/aab0c5aad3f41ba92996db5a74e0d19abfa92500/changelog/@unreleased/pr-227.v2.yml#L3)):*
<details><summary><i>Previously, nmiyake (Nick Miyake) wrote…</i></summary><blockquote>

Will this entry/description make sense as part of the over-all release notes of v2.0?
</blockquote></details>

Yeah, good call. Updated with full migration details.

**Review by nmiyake — COMMENTED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/palantir/witchcraft-go-server/227)* status: all files reviewed, 1 unresolved discussion (waiting on @asanderson15)

___
*[go.mod, line 18 at r1](https://reviewable.io/reviews/palantir/witchcraft-go-server/227#-MLmt2Su5O35XvQD3bE4-r1-18:-MLnAn6VBSb-KzCuIXu0:b-gs1kyq) ([raw file](https://github.com/palantir/witchcraft-go-server/blob/aab0c5aad3f41ba92996db5a74e0d19abfa92500/go.mod#L18)):*
<details><summary><i>Previously, asanderson15 (Adam Anderson) wrote…</i></summary><blockquote>

We should release a tagged version of WGH and bump to depend on that before merging this.
</blockquote></details>

Will wait on this to update before approving

**Review by nmiyake — APPROVED** [review]
> Reviewed 3 of 3 files at r3.
> *[Reviewable](https://reviewable.io/reviews/palantir/witchcraft-go-server/227)* status: all files reviewed, 1 unresolved discussion (waiting on @asanderson15)

**Pull request merged**